### PR TITLE
Sort ddoc table of functions in range package

### PR DESCRIPTION
**Pull request by kuettler**
A search on a sorted list is so much nicer, especially for humans.

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -16,55 +16,38 @@ $(I On Iteration)).
...
 The remainder of this module provides a rich set of _range creation and
 composition templates that let you construct new ranges out of existing ranges:
 
 $(BOOKTABLE ,
-    $(TR $(TD $(D $(LREF retro)))
-        $(TD Iterates a bidirectional _range backwards.
-    ))
-    $(TR $(TD $(D $(LREF stride)))
-        $(TD Iterates a _range with stride $(I n).
+    $(TR $(TD $(D $(LREF NullSink)))
```
Comment (quickfur) [inline std/range/package.d]: I don't think this should be at the top; the sort order should be case-insensitive.

**Comment by quickfur** [comment]
Other than that, LGTM. Thanks!

File: std/range/package.d (hunk 1 above)
Comment (kuettler) [inline std/range/package.d]: Done.

File: std/range/package.d [hunk 2]
```diff
@@ -16,55 +16,35 @@ $(I On Iteration)).
 
 Submodules:
 
-This module has a few submodules:
+This module has a two submodules:
```
Comment (quickfur) [inline std/range/package.d]: Grammar: "... has two submodules", not "has a two submodules".

**Comment by quickfur** [comment]
Whoa, another one ready to merge but sleeping for 26 days now? Time to merge! We'll clean up any messes later, if necessary.

**Comment by quickfur** [comment]
Auto-merge toggled on

**Pull request merged**